.circleci/config.yml

```diff
@@ -449,12 +449,15 @@ jobs:
       - run:
           name: Save Yarn version
           command: yarn --version > /tmp/YARN_VERSION
+      - run:
+          name: Save Foundry version
+          command: node -e "process.stdout.write(require('./package.json').foundry)" > /tmp/FOUNDRY_VERSION
```

Comment:
So I've added a new field to our `package.json`: `foundry`. It contains the release we use in the installation process, as well as in this cache process.

I don't like it.

Please please please recommend a better way <3